component-samples/demo/owner/README.md

```diff
@@ -68,13 +68,13 @@ Owner runtime arguments:
 
   Path to the Owner keystore file containing the owner's keys.
 
-  Default value: ./owner_keystore.p12
+  The owner keystore file and path value is generated using keys_gen.sh script and is stored in creds.env file. [Read more](https://github.com/secure-device-onboard/pri-fidoiot/tree/master/component-samples/demo#preparing-credentials-for-components) about the key generation script here.
```

Comment:
For uniformity with other component's same section, remove word 'owner' here and at property owner_ssl_keystore

Reply:
Done.